# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../procps/procps-3.2.8-make-3.82.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix build using make >= 3.82

--- procps-3.2.8/Makefile.orig	2012-10-21 02:26:06.961308206 +0200
+++ procps-3.2.8/Makefile	2012-10-21 02:26:26.205720768 +0200
@@ -8,7 +8,7 @@
 # hasn't grown big enough to need something complicated, and the
 # dependency tracking files are an ugly annoyance.
 #
-# This file includes */module.mk files which add on to variables:
+# This file includes proc/module.mk ps/module.mk files which add on to variables:
 # FOO += bar/baz
 #
 #
@@ -174,7 +174,7 @@
 # want this rule first, use := on ALL, and ALL not filled in yet
 all: do_all
 
--include */module.mk
+-include proc/module.mk ps/module.mk
 
 do_all:    $(ALL)
 
